packages/stokhos/src/sacado/kokkos/vector/Stokhos_MP_Vector_MaskTraits.hpp

```diff
@@ -73,6 +74,8 @@ template<typename scalar> class Mask
 
     public:
         Mask(){
+            for(auto i=0; i<size; ++i)
```

Comment:
I have mixed feelings about `auto` indices -- it's correct here, because literal `0` is an `int`, but in general it could lead to signed / unsigned comparison warnings or (worse) overflow, for example if `size` is a `size_t`.